### PR TITLE
System Information

### PR DESCRIPTION
**Pull request by brianteeman**
We had the db version and the db collation but not the type in the System Information tab of the System information admin menu

This PR adds the database type eg postgresql, mysql etc

### CODE REVIEW

**Comment by sandewt** [comment]
I have tested this item :white_check_mark: successfully on 51ad8c7b2d9fed470cc60f52f3c7be9acd576486<br /><br />Database Type 	mysql 
Windows 10
XAMPP
PHP 7.1.9 <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19764">issues.joomla.org/tracker/joomla-cms/19764</a>.</sub>

**Comment by sandewt** [comment]
Test result.

screen shot 2018-02-23 at 14 49 49<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19764">issues.joomla.org/tracker/joomla-cms/19764</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 51ad8c7b2d9fed470cc60f52f3c7be9acd576486<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19764">issues.joomla.org/tracker/joomla-cms/19764</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19764">issues.joomla.org/tracker/joomla-cms/19764</a>.</sub>

File: administrator/components/com_admin/models/sysinfo.php [hunk 1]
```diff
@@ -309,6 +309,7 @@ public function &getInfo()
 		$this->info = array(
 			'php'                   => php_uname(),
 			'dbversion'             => $db->getVersion(),
+			'serverType'		=> $db->getServerType(),
```
Comment (Quy) [inline administrator/components/com_admin/models/sysinfo.php]: Maybe change it to `dbtype` and move it above `dbversion`. Server type could be interpreted as Apache if just reading the code.

Comment (brianteeman) [reply administrator/components/com_admin/models/sysinfo.php]: I went for matching the function but happy to change it do dbserver

Comment (Quy) [reply administrator/components/com_admin/models/sysinfo.php]: `dbserver` is better.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 7bfdf24c16220feab36dc6386d2644a4682b38d5<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19764">issues.joomla.org/tracker/joomla-cms/19764</a>.</sub>

**Comment by sandewt** [comment]
I have tested this item :white_check_mark: successfully on 7bfdf24c16220feab36dc6386d2644a4682b38d5<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19764">issues.joomla.org/tracker/joomla-cms/19764</a>.</sub>

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**